workers/common/src/main/kotlin/common/OrtServerFileArchiveStorage.kt

```diff
@@ -58,15 +59,16 @@ class OrtServerFileArchiveStorage(
             }
     }
 
-    override fun getData(provenance: KnownProvenance): InputStream? {
+    override fun getData(provenance: KnownProvenance): InputStream? = runBlocking {
```

Comment:
Not sure whether using `runBlocking` here is a good solution. From what I have read, this function should only be used in tests and for the main entry point.
Would it be an alternative to create a new `CoroutineScope` in this class and use it to launch the suspend functions?

Reply:
The problem with `launch` is that you get a `Job`. Should we then `await` on it ?

Reply:
You are right, it is probably not worth the effort. I was reading [this article](https://discuss.kotlinlang.org/t/globalscope-and-runblocking-should-not-be-used-so-how-do-you-start-a-coroutine/24004/3), and while statements are a bit contradicting IMHO, it could be interpreted that `runBlocking` was appropriate for our use case.

Reply:
I will remember that from your link:

> runBlocking should not be used from inside the coroutine, but it is a normal way to enter the coroutine world.